automation/taskcluster/lib/tasks.py

```diff
@@ -441,15 +441,7 @@ def craft_raptor_signing_task(
         self, assemble_task_id, variant, is_staging,
     ):
         staging_prefix = '.staging' if is_staging else ''
-        routes = [
-            "index.project.mobile.fenix.v2{}.performance-test.{}.{}.{}.latest.{}".format(
-                staging_prefix, self.date.year, self.date.month, self.date.day, variant.abi
-            ),
-            "index.project.mobile.fenix.v2{}.performance-test.{}.{}.{}.revision.{}.{}".format(
-                staging_prefix, self.date.year, self.date.month, self.date.day, self.commit, variant.abi
-            ),
-            "index.project.mobile.fenix.v2{}.performance-test.latest.{}".format(staging_prefix, variant.abi),
-        ]
+        routes = []
```

Comment:
Same here

Reply:
I reverted the changes in this commit:
https://github.com/mozilla-mobile/fenix/pull/4430/commits/55a76b8aa47ca771022ed3cdafd5c999e80b6e49